cswinrt/code_writers.h

```diff
@@ -4410,7 +4425,7 @@ IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
 %}
 )",
             bind<write_winrt_attribute>(type),
-            bind<write_custom_attributes>(type),
+            bind<write_custom_attributes>(type, false),
```

Comment:
Any reason we don't do it for attributes?

Reply:
not supported (C# compilation error - I tried every type I could)